### PR TITLE
Upgrade to stretch

### PR DESCRIPTION
**Pull request by stefanor**
And pick up some loose bits that were lying around in production checkouts...

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: doorservice.service [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Description=doorservice
+After=network.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Restart=always
+
+ExecStart=/home/pi/doorservice/virtualenv/bin/python run.py
```
Comment (zoidbergwill) [inline doorservice.service]: We could use the shiny [`flask run` CLI](http://flask.pocoo.org/docs/1.0/cli/) too?

Comment (zoidbergwill) [reply doorservice.service]: Oh wait, maybe not with the Flask version.

Comment (stefanor) [reply doorservice.service]: We could upgrade it...

**Comment by stefanor** [comment]
:bell: @stormf

**Pull request merged**